src/Schedule.h

```diff
@@ -232,10 +241,21 @@ class Schedule {
      * true. See \ref Func::specialize */
     // @{
     const std::vector<Specialization> &specializations() const;
+    std::vector<Specialization> &specializations();
```

Comment:
yes, need this to substitute in the wrapper func after deep-copy. Per our discussion, removed this mutable accessor and added mutate() method instead.